### ISSUE TITLE
Some problem about softmax

### ISSUE THREAD

**Issue by IvenHsu** [tensorflow/tensorflow#1565]
Hi I have problem about the softmax

I use the Softmax_with_loss to train the VOC2007 image data set.
and image have been normal between -1,1.
but it have two experiment result.

network have 5 conv weight and 3 fully-connect
only train 2 image and batch size 1
learning rate = 0.005

In case 1:
initialize weight and biases arg stddev = 0.01
The loss continued to decline on very small value

In case 2
initialize weight and biases arg 0.1
The loss decline to 0.6
but it doesn't continued to low
and it can't effective prediction the training data

Is it right for training?
The image number and batch size are affect the initialize weight and biases?

**Comment by martinwicke** [comment]
This is a question better suited for stackoverflow -- please ask there.
